docs/_static/nuts-auth.yaml

```diff
@@ -644,9 +582,12 @@ components:
             As per rfc7523 https://tools.ietf.org/html/rfc7523>, the aud must be the
             token endpoint. This can be taken from the Nuts registry.
           example: "https://target_token_endpoint"
-        uid:
+        usi:
           type: string
           description: Jwt encoded user identity.
+        osi:
+          type: string
+          description: encoded ops signature.
```

Comment:
I feel like this description could be a bit more descriptive

Reply:
yes, when we know what is actually going to be in there ;)